translate/v23tov30/v23tov30.go

```diff
@@ -347,12 +348,50 @@ func translateFilesystems(fss []old.Filesystem, m map[string]string) (ret []type
 		if f.Mount == nil {
 			f.Mount = &old.Mount{}
 		}
+
+		wipe := util.BoolP(f.Mount.WipeFilesystem)
+		label := f.Mount.Label
+		uuid := f.Mount.UUID
+
+		// If `wipe` is set to `false` but we have a `"create": {...}` section, we try
+		// to convert it.
+		if wipe == nil && f.Mount.Create != nil {
```

Comment:
Mm, it's done previously:
```go
wipe := util.BoolP(f.Mount.WipeFilesystem)

if wipe == nil ...
```

Doing this way, we're sure that `WipeFilesystem` is not set since `util.BoolP` returns `nil` if non-set.

Reply:
Right.  I'm saying, shouldn't the test be:

```suggestion
		if f.Mount.Create != nil {
```

If the user specified `Create` options, I assume we always want to respect them.